cmd/skaffold/app/cmd/init.go

```diff
@@ -106,7 +106,7 @@ func doInit(out io.Writer) error {
 
 	for _, file := range potentialConfigs {
 		if !force {
-			config, err := schema.ParseConfig(file, true)
+			config, err := schema.ParseConfig(file, false)
```

Comment:
I'm not sure I follow this one. Why wouldn't we want to upgrade?

Reply:
In fact, here, we don't really care. We just want to check that those are valid config files.